### DISCUSSION TITLE
How do you get the claimblocks a user has on a sidebar scoreboard?

### DISCUSSION THREAD

**Discussion by jonathanVanH** [GriefPrevention/GriefPrevention#881] [Q&A] [answered]
Hello! 

I've seen some survival servers use scoreboards that have the griefprevention claimblocks displayed on it and i was wondering how they added it. I imported the plugin in Maven but i can't find the methods used to return the claimblocks a player has left.

**Comment by Jikoo** [comment, marked as answer]
`PlayerData#getRemainingClaimBlocks`. You can get a `PlayerData` instance from the `DataStore`.

I know you're looking to implement this yourself, but PAPI [has placeholders for GP already](https://github.com/PlaceholderAPI/PlaceholderAPI/wiki/Placeholders#griefprevention); servers you've seen claim block counts on are likely using it in conjunction with a compatible scoreboard plugin.
